### ISSUE TITLE
Trumbowyg editor modals display in light mode while dark mode is active

### ISSUE THREAD

**Issue by mdameer** [OrchardCMS/OrchardCore#16362]
### Describe the bug
When dark mode is enabled, the Trumbowyg editor modals do not follow the dark theme and instead display in light mode. This inconsistency affects the user experience.

![image](https://github.com/OrchardCMS/OrchardCore/assets/2022320/7f4c99e3-31ee-46d7-b0d9-fd5c235ee510)

### Orchard Core version
Latest code from main

### To Reproduce
1. Enable dark mode.
2. Navigate to any content item that uses the Trumbowyg editor.
3. Open the Insert link modal.
4. Observe that the modal displays in light mode despite active dark mode.

### Expected behavior
The Trumbowyg editor modal should adapt to the active theme and display in dark mode when dark mode is enabled.

**Comment by hishamco** [comment]
I think there's a similar issue already there

**Comment by mdameer** [comment]
@hishamco I found another issue regarding the Trumbowyg editor theme (#16137). I have a draft PR (#16356) that will address both, so for a clear history of resolved issues I created this one, also to discuss the ability to use the default Trumbowyg editor theme which already supports light/dark modes, and why we have `_trumbowyg.scss`? do we need it?

Orchard core
![image](https://github.com/OrchardCMS/OrchardCore/assets/2022320/e0f34041-ee23-4b5d-a928-a47724f22ee6)

Trumbowyg default theme
![image](https://github.com/OrchardCMS/OrchardCore/assets/2022320/8e9aaef1-ea9e-433c-8623-a202632ddc37)
